lightning/src/ln/channelmanager.rs

```diff
@@ -749,6 +749,11 @@ pub struct ChannelManager<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref,
 	/// [fake scids]: crate::util::scid_utils::fake_scid
 	fake_scid_rand_bytes: [u8; 32],
 
+	/// When we send payment probes, we generate the [`PaymentHash`] based on this cookie secret
+	/// and a random [`PaymentId`]. This allows us to discern probes from real payments, without
+	/// keeping additional state.
+	probing_cookie_secret: [u8; 32],
```

Comment:
To keep concepts nice and separate, I now introduced a new field `probing_cookie_secret`. However, if this was wanted, we could also potentially recycle the above `fake_scid_rand_bytes`.